src/task-actions.ts

```diff
@@ -338,6 +338,13 @@ async function handleContractVerification(
 
     const bodyFormData = new FormData();
     bodyFormData.append("compiler-version", args.compilerVersion);
+    let accountContract;
+    if (args.accountContract && args.accountContract === true) {
```

Comment:
This doesn't look good:
- it's redundant to check both `args.accountContract` and `args.accountContract === true`, the latter should be enough
- this condition will never be met because the user cannot specify a boolean value through the CLI, right?
- this could as well be achieved with a ternary operator
- this if-else structure means that anything other than `true` will be treated as false, do we want that?

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/172/commits/c8ecc43a6a76744cc346bba1728c66a49caa916a